zebra-scanner/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "zebra-scanner"
+version = "1.0.0-beta.30"
+authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Blockchain scanner for the Zebra Zcash node"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
+edition = "2021"
+
+readme = "../README.md"
+homepage = "https://zfnd.org/zebra/"
+# crates.io is limited to 5 keywords and categories
+keywords = ["zebra", "zcash"]
+# Must be one of <https://crates.io/category_slugs>
+categories = ["cryptography::cryptocurrencies"]
+
+[features]
+
+# Production features that activate extra dependencies, or extra features in dependencies
+
+[dependencies]
+zcash_client_backend = "0.10.0-rc.1"
+zcash_primitives = "=0.13.0-rc.1"
+zcash_note_encryption = "0.4.0"
+
+rand = "0.8.5"
+
+bls12_381 = "0.8.0"
+jubjub = "0.10.0"
+ff = "0.13.0"
+group = "0.13.0"
+
+tokio = { version = "1.33.0", features = ["full", "tracing", "test-util"] }
```

Comment:
We avoid depending on all of `tokio`'s features in production code. And there's no need to depend on test features in production builds.

Here's how to fix this:

In production (here):
```toml
tokio = { version = "1.33.0", features = ["tracing"] }
```

And whatever else is needed from this list:
https://docs.rs/tokio/latest/tokio/#feature-flags

We usually have something like:
https://github.com/ZcashFoundation/zebra/blob/bb7eeb6f76a91d8e6659891ff6d8bd0e055d2b74/zebra-consensus/Cargo.toml#L56

And then in `[dev-dependencies]`:
```toml
tokio = { version = "1.33.0", features = ["full", "tracing", "test-util"] }
```

Reply:
Replaced by https://github.com/ZcashFoundation/zebra/pull/7758/files#r1370890516